### ISSUE TITLE
Wrong receipt in Subscriber for Inscriber Silicon Press

### ISSUE THREAD

**Issue by uhloin** [AppliedEnergistics/Applied-Energistics-2#5796]
In Subscriber receipt:
Block of Iron -> Inscriber Silicon Press -> Inscriber Silicon Press
This has not sence because spends Press and creates Press again.
Should be:
Inscriber Silicon Press -> Block of Iron -> (new) Inscriber Silicon Press

Envoironment:
AE version: 10.0.0-alpha.6 [FORGE]
Minecraft 1.18.1, 
Forge 39.0.0,
https://www.curseforge.com/minecraft/mc-mods/applied-energistics-2/files/3566597

**Comment by Technici4n** [comment]
Can't reproduce this on fabric:
![image](https://user-images.githubusercontent.com/13494793/146620337-04cb0659-a22e-4ff6-a80f-7b88037cea5b.png)

Will need checking on the forge branch.

**Comment by shartte** [comment]
Yes, this is indeed a leftover bug from Forge 1.17. We'll fix it in the next release.

**Comment by shartte** [comment]
Will be fixed in 10.0.0-alpha.7 for Forge
